comments/framing_for_selection.md

```diff
@@ -0,0 +1,43 @@
+# Framing for selection
+
+Thinking about how selection and changes to selection are shown.
+
+### Selection
+
+There can be one 'selected' item. This must be something in the model identified by an IRI, and then appears in the 'hash' portion of the url.
+
+The selected item panel reacts to the change of selection, and various aspects of the map presentation do as well - labels for that item have a (subtle, dark) border, edges may be shown, and so on.
+
+
+### Framing
+
+How the browser window presents the map pane. This is primarily the centre point and resolution of the map, both of which are adaptive based on screen size and shape. Experimentally, this can also include shading of an area related to the item, or some other way of marking map content.
+
+
+## Expectations
+
+The presented selection should be clear and meaningful. That means it should ideally have visible on screen all of the content related to the selection, and some of the surrounding context from the environment, such as the parent, and siblings. Content should not be obscured by other elements of the platform. For container-type items, what is contained within them should be clear, even if all descendants are not shown at that detail level. There's an intersection between map design, ontology, and behaviours derived from the model.
+
+
+## Existing behaviours
+
+Currently there are three distinct cases to consider:
+* New map load with selection set
+* Select new item from link outside the map
```

Comment:
sorry but map and map can mean different. You mean if I just load a URL from email or I'm on Org map but need to go to Activity map